spec/services/courses_service_spec.rb

```diff
@@ -5,18 +5,28 @@
 
   subject { described_class.new storyblok_client }
 
+  let(:latest_cache_version) { '123' }
```

Comment:
What is the 123 here?

Reply:
This is just a test / mocked value for the spec, which is used in the `allow(…)` bit below to mock an API call.